modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -7250,13 +7250,40 @@ public void testPostCustomObjectEntryWithNestedCustomObjectEntriesInManyToOneRel
 			_objectDefinition1, _objectDefinition2, TestPropsValues.getUserId(),
 			ObjectRelationshipConstants.TYPE_ONE_TO_MANY);
 
+		_objectFieldLocalService.updateRequired(
+			_objectRelationship1.getObjectFieldId2(), true);
+
 		JSONObject objectEntryJSONObject = JSONUtil.put(
 			_objectRelationship1.getName(),
 			JSONFactoryUtil.createJSONObject(
 				JSONUtil.put(
 					_OBJECT_FIELD_NAME_1, _NEW_OBJECT_FIELD_VALUE_1
 				).put(
-					"externalReferenceCode", _ERC_VALUE_1
+					"externalReferenceCode", ""
+				).toString()));
+
+		try (LogCapture logCapture = LoggerTestUtil.configureLog4JLogger(
+				"com.liferay.portal.vulcan.internal.jaxrs.exception.mapper." +
+					"WebApplicationExceptionMapper",
+				LoggerTestUtil.ERROR)) {
+
+			Assert.assertEquals(
+				400,
+				HTTPTestUtil.invokeToHttpCode(
+					objectEntryJSONObject.toString(),
+					_objectDefinition2.getRESTContextPath(), Http.Method.POST));
+		}
+
+		_objectFieldLocalService.updateRequired(
+			_objectRelationship1.getObjectFieldId2(), false);
+
+		objectEntryJSONObject = JSONUtil.put(
```

Comment:
Do we need to override this variable? Doesn't it have the same value it already has?